config/config.go

```diff
@@ -797,6 +797,9 @@ type Experimental struct {
 	AllowsExpressionIndex bool `toml:"allow-expression-index" json:"allow-expression-index"`
 	// Whether enable charset feature.
 	EnableNewCharset bool `toml:"enable-new-charset" json:"-"`
+	// Whether enable foreign key feature.
+	// TODO(crazycs520): remove this after foreign key GA.
+	EnableForeignKey bool `toml:"enable-foreign-key" json:"enable-foreign-key"`
```

Comment:
Can you add this as a system variable instead? This makes it easier to test globally.

Reply:
great, done.

Reply:
@crazycs520 I think we don't need to have them both?

Reply:
great, done.